Cargo.toml

```diff
@@ -10,6 +10,8 @@ repository = "https://github.com/nagisa/rust_libloading/"
 documentation = "https://nagisa.github.io/rust_libloading/"
 
 [dependencies]
+lazy_static = "0.2"
+
+[target.'cfg(windows)'.dependencies]
```

Comment:
`target.cfg(...).dependencies` is supported starting in cargo 0.9.0 (Rust 1.8.0).